src/main/resources/META-INF/persistence.xml

```diff
@@ -6,6 +6,15 @@
 
 	<persistence-unit name="default" transaction-type="RESOURCE_LOCAL">
 		<provider>org.hibernate.ejb.HibernatePersistence</provider>
+		<class>nl.tudelft.ewi.devhub.server.database.entities.BuildResult</class>
```

Comment:
There shouldn't be a need to manually register classes in the current setup. Have you tested without this?